Gopkg.toml

```diff
@@ -26,7 +26,7 @@
 
 [[constraint]]
   name = "github.com/IBM/go-sdk-core"
-  version = "0.4.2"
+  version = "0.5.0"
```

Comment:
no 1.0.0 yet? ;)

Reply:
We can probably declare the 1.0.0 release of the core when we deliver the actual major release.